src/components/AppProvider/AppProvider.tsx

```diff
@@ -1,8 +1,16 @@
-import React from 'react';
+import React, { Suspense } from 'react';
+import { FirebaseAppProvider } from 'reactfire';
 import { ThemeDecorator } from 'utils/theme';
+import { firebaseConfig } from 'utils/firebase';
 
 const AppProvider: React.FC = ({ children }) => {
-  return <ThemeDecorator>{children}</ThemeDecorator>;
+  return (
+    <ThemeDecorator>
+      <FirebaseAppProvider firebaseConfig={firebaseConfig}>
+        <Suspense fallback={<p>loading status...</p>}>{children}</Suspense>
```

Comment:
Add creating a loading component or multiple loading components to the to-do list.

Reply:
I created a board in `Projects` and moved the to-dos there